### PR TITLE
refactor aria-labels for preferred provider learn more buttons

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19780

## Testing done
Dev tested.

## Screenshots
No screenshots needed.

## Acceptance criteria
- [ ]  The "Learn More" link aria label text is the text specified in supporting document # 1.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]
Maybe make the constants file more specific like ariaLabelConstants.js. Generally, our constants in js are just using camelCase not SCREAMING_SNAKE_CASE.

File: src/applications/gi/constants.js [hunk 1]
```diff
@@ -1,2 +1,6 @@
 // WAIT_INTERVAL is in milliseconds.
 export const WAIT_INTERVAL = 333;
+
+export const ARIA_LABELS = Object.freeze({
```
Comment (shawkey-daniel) [inline src/applications/gi/constants.js]: Can we name it ariaLabels?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/gi/constants.js (hunk 1 above)
Comment (shawkey-daniel) [reply src/applications/gi/constants.js]: I would like to change waitInterval too, but not necessary for this change.

Comment (hughes-dustin) [reply src/applications/gi/constants.js]: Sure thing.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Pull request merged**